app/views/idv/doc_auth/_start_over_or_cancel.html.erb

```diff
@@ -1,7 +1,7 @@
 <br/>
 <%= button_to(
   t('doc_auth.buttons.start_over'),
-  idv_doc_auth_step_path(:reset),
+  idv_doc_auth_step_path(step: :reset),
```

Comment:
this is optional right? Just to make things clearer?

Reply:
Yep, I actually needed to stick it in for the view to render w/o a controller.